pandas/core/dtypes/cast.py

```diff
@@ -333,7 +333,7 @@ def maybe_promote(dtype, fill_value=np.nan):
     return dtype, fill_value
 
 
-def infer_dtype_from_scalar(val, pandas_dtype=False):
+def infer_dtype_from_scalar(val, pandas_dtype=False, use_datetimetz=True):
```

Comment:
you don't need this extra parameter, instead you can pass ``pandas_dtype=True`` . what is the issue?